lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -66,10 +66,34 @@ blockchain_configuration:
 
 # The initial deposits present in the blockchain.
 #
-# It can be UTxO addresses or account (if allow_account_creation is set).
+# The last 10 addresses come from 10 different wallets.
+# For the integration test's faucets to be of similar shape
+# with the one used with the http-bridge,
+# we need each wallet to have 10 UTxO available, hence the value for each
+# wallet is 10x bigger than what is ascribed to each UTxO.
```

Comment:
> hence the value for each wallet is 10x bigger than what is ascribed to each UTxO.

Making the value bigger doesn't make more UTxO available :wink: UTxOs are like bank notes, here we don't want a 500 EUR note, but we do want 10 x 50 EUR notes! So, instead of 

```yaml
  - address: ta1svz5hw3leda47fy3scdku2ut7z3ljftumuha7sw2ttmswtnhcu8n5nm0ynp
    value: 1000000000000
```

We should have 

```yaml
  - address: ta1svz5hw3leda47fy3scdku2ut7z3ljftumuha7sw2ttmswtnhcu8n5nm0ynp
    value: 100000000000
  - address: ta1svz5hw3leda47fy3scdku2ut7z3ljftumuha7sw2ttmswtnhcu8n5nm0ynp
    value: 100000000000
.
. ( x 10 in total)
.
  - address: ta1svz5hw3leda47fy3scdku2ut7z3ljftumuha7sw2ttmswtnhcu8n5nm0ynp
    value: 100000000000
```

And this for each address!

Reply:
Done in: 94ff494ec7b79819e66696c83a1755a71c1d3c54